types/fulu/block_contents.yaml

```diff
@@ -0,0 +1,24 @@
+Fulu:
+  BlockContents:
+    type: object
+    description: "The required object for block production according to the Fulu CL spec."
+    required: [block, cell_proofs, blobs]
+    properties:
+      block:
+        $ref: "./block.yaml#/Fulu/BeaconBlock"
+      cell_proofs:
```

Comment:
How do people feel about keeping the `kzg_proofs` field name? Not clear to me how much we gain from this rename, and given the beacon APIs are fork aware, this change doesn't feel necessary.